HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1800,6 +1804,251 @@ def self.disaggregate_fan_or_pump(model, fan_or_pump, htg_object, clg_object, ba
     end
   end
 
+  def self.apply_max_power_EMS(model, runner, hpxml_bldg, air_loop_unitary, control_zone, heating_system, htg_supp_coil = nil, clg_coil, htg_coil, schedules_file)
+    if not schedules_file.nil?
+      max_pow_ratio_sch = schedules_file.create_schedule_file(model, col_name: SchedulesFile::ColumnHVACMaximumPowerRatio, schedule_type_limits_name: Constants.ScheduleTypeLimitsFraction)
+      # Not allowed with unit multiplier for now
+      if not max_pow_ratio_sch.nil?
```

Comment:
Is this the right way to trigger disallowing unit multipliers? I'd have thought you'd have to check for a multiplier here, not just if there's a schedule file and the max power column isn't there. Can we more directly look at the unit multiplier here?

Reply:
Yeah probably you're missing the "if" statement in the line1812, that's the checking for multiplier.

Reply:
Yup I sure did! Sorry about that, makes sense now.